classes/indoorFloor.tsx

```diff
@@ -39,59 +45,174 @@ class IndoorFloor {
         this.initialHeightPosition = newInitialHeightPosition;
     }
 
-    setImageWidth(newImageWidth: number){
+    setImageWidth (newImageWidth: number) {
         this.imageWidth = newImageWidth;
         const ratio = this.floorData.imageWidthPX / this.floorData.imageHeightPX;
-        this.imageHeight = newImageWidth*ratio;
+        this.imageHeight = newImageWidth * ratio;
     }
 
-    showFloorImage(){
-        return(
+    setFloorImageStyle():ImageStyle{
+        return{
+            width: this.imageWidth, height: this.imageHeight, position: "absolute", top: this.initialHeightPosition, left: this.initialWidthPosition, zIndex: 4
+        }
+    }
+    showFloorImage() {
+        return (
             <Image
                 resizeMode={'cover'}
-                style={{width: this.imageWidth, height: this.imageHeight, position: "absolute", top: this.initialHeightPosition, left: this.initialWidthPosition, zIndex: 4}}
+                style={this.setFloorImageStyle()}
                 source={this.floorData.floorImage}
             />
         )
     }
+    // This function has been commented out because it's out of use. It could be useful when updating the infloormap data in the future though
+    // Concerning the walkables paths
+
```

Comment:
dont comment out the function as it looks like it could be deleted. The comment you left above is good to keep though